internal/server/http/server.go

```diff
@@ -170,9 +174,19 @@ func (s *Server) addImposterHandler(imposters []Imposter) {
 				r.Queries(k, v)
 			}
 		}
+		log.WithFields(imposter.LogFields()).Debugln("imposter loaded")
+	}
+}
+
+func (s *Server) defaultNotFoundHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		log.WithFields(killgrave.LogFieldsFromRequest(r)).Debugf("Request didn't match any imposter, and proxyMode is %v", s.proxy.mode)
+		w.WriteHeader(http.StatusNotFound)
+		w.Write([]byte("404 page not found\n"))
 	}
 }
 
+// not used?
```

Comment:
also, @joanlopez , is the method ever used? since we configure `s.router.NotFoundHandler` i would assume that already acts as a catch all